### ISSUE TITLE
Inconsistent Sizing on \overleftarrow

### ISSUE THREAD

**Issue by matoushs** [mathjax/MathJax#2135]
### Issue Summary
The over left arrow is considerably smaller than the over right arrow with HTML-CSS renderer. This TeX code: ```$\overrightarrow{R} \overleftarrow{R} \leftarrow \rightarrow$```, renders as: 

![image](https://user-images.githubusercontent.com/15343188/55462242-931d5680-55f6-11e9-8086-d329cabf0f93.png)

The MML code shows stretchy=false on the left arrow:
```
<math xmlns="http://www.w3.org/1998/Math/MathML">
  <mover>
    <mi>R</mi>
    <mo>&#x2192;<!-- → --></mo>
  </mover>
  <mover>
    <mi>R</mi>
    <mo>&#x2190;<!-- ← --></mo>
  </mover>
  <mo stretchy="false">&#x2190;<!-- ← -->&#x2192;<!-- → --></mo>
</math>
```

After reading #1470 I added the STIX font to my configuration file: 

   "HTML-CSS": {
        preferredFont: "STIX"
    }

The font is STIX, but the arrows are still different. 

![mathjax-ss](https://user-images.githubusercontent.com/15343188/55461991-1ab69580-55f6-11e9-9be5-76ed7ec94e65.png)

### Technical details:

* MathJax Version: 2.3 (latest commit: f3aaf3a2a3e964df2770dc4aaaa9c87ce5f47e2c)
* Client OS: Mac OS X 10.14.3
* Browser: Firefox Developer - Aurora

**Comment by dpvc** [comment]
Thanks for the report.  You are right, the data for local STIX fonts is missing the stretchy delimiter data for the combining left arrow, as your original title suggested.  If you add

```
<script type="tet/x-mathjax-config">
MathJax.Hub.Register.LoadHook("[MathJax]/jax/output/HTML-CSS/fonts/STIX/fontdata.js", function () {
  MathJax.OutputJax["HTML-CSS"].FONTDATA.DELIMITERS[0x20D6] = {alias: 0x2190, dir: "H"};
});
</script>
```

to your page just **before** the script that loads MathJax.js itself, that should resolve the issue until the file is fixed.  It would also be possible to put this into a configuration file that you load in the same script that loads MathJax.js (by adding its full URL to the list of `config=` files and ending the file with a `MathJax.Ajax.loadComplete()` that is passed the same URL).

**Comment by dpvc** [comment]
In testsuite:

MathMLToDisplay/Presentation/ScriptsAndLimits/issue2135.html

**Comment by Shijie97** [comment]
> Thanks for the report. You are right, the data for local STIX fonts is missing the stretchy delimiter data for the combining left arrow, as your original title suggested. If you add
> 
> ```
> <script type="tet/x-mathjax-config">
> MathJax.Hub.Register.LoadHook("[MathJax]/jax/output/HTML-CSS/fonts/STIX/fontdata.js", function () {
>   MathJax.OutputJax["HTML-CSS"].FONTDATA.DELIMITERS[0x20D6] = {alias: 0x2190, dir: "H"};
> });
> </script>
> ```
> 
> to your page just **before** the script that loads MathJax.js itself, that should resolve the issue until the file is fixed. It would also be possible to put this into a configuration file that you load in the same script that loads MathJax.js (by adding its full URL to the list of `config=` files and ending the file with a `MathJax.Ajax.loadComplete()` that is passed the same URL).

Hello, I'd love to use this plan in Tex Studio, what should I do?

**Comment by dpvc** [comment]
This issue was fixed in v2.7.8 of MathJax, so the easiest solution is to update to 2.7.8 or v2.7.9 (the highest v2 version).  Then you will get the fix automatically.
